pkgs/development/tools/rust/rust-analyzer/update.sh

```diff
@@ -51,6 +51,10 @@ build_deps="../../../../misc/vscode-extensions/rust-analyzer/build-deps"
 jq '{ name, version, dependencies: (.dependencies + .devDependencies) }' "$node_src/package.json" \
     >"$build_deps/package.json.new"
 
+# FIXME: Workaround for https://github.com/NixOS/nixpkgs/issues/100077
```

Comment:
Could you also add a comment explaining how this workaround solves the problem?

Reply:
Sorry but I don't know why it works either. :disappointed:

`node2nix` generates different overrides on this package when explicitly specified, and the default one makes npm unhappy.
See also https://github.com/svanderburg/node2nix/issues/171#issuecomment-707289163

Reply:
I see. In that case, could you also add a link to https://github.com/svanderburg/node2nix/issues/171#issuecomment-707289163 in the comment?